### DISCUSSION TITLE
docker版本如何运行命令行获取refresh token？

### DISCUSSION THREAD

**Discussion by 404gods** [messense/aliyundrive-webdav#744] [Q&A] [answered]
README.md上说最新的V2要用命令行运行 aliyundrive-webdav qr login 扫码授权后会输出 refresh token，请问docker版本如何运行命令行获取refresh token？还是说仍然用网页获取refresh token？

**Comment by tlze** [comment, marked as answer]
```
docker run --rm -it messense/aliyundrive-webdav /bin/sh
aliyundrive-webdav qr login
```

**Reply by messense** [reply]
增加了在线工具获取：https://github.com/messense/aliyundrive-webdav#%E8%8E%B7%E5%8F%96-refresh-token
